derive/src/multihash.rs

```diff
@@ -314,7 +315,13 @@ fn error_alloc_size(hashes: &[Hash], expected_alloc_size_type: &syn::Type) {
 }
 
 pub fn multihash(s: Structure) -> TokenStream {
-    let mh_crate = utils::use_crate("multihash");
+    let mh_crate = match utils::use_crate("multihash") {
+        Ok(ident) => ident,
+        Err(e) => {
+            let err = Error::new(Span::call_site(), e).to_compile_error();
```

Comment:
As we have different errors in this file, I find it clearer if you would use `syn::Error` instead of just `Error` here. Can you please change that?

Reply:
Done